app/routes/circulars.$circularId.($version)/rehypeAutolinkLiteral.ts

```diff
@@ -20,7 +20,11 @@ const regexp = new RegExp(
 function autolinkLiteral(tree: Parameters<typeof findAndReplace>[0]) {
   findAndReplace(
     tree,
-    [regexp, (href: string) => h('a', { rel: 'external', href }, href)],
+    [
+      regexp,
+      (href: string) =>
+        h('a', { rel: 'noreferrer', href, target: '_blank' }, href),
```

Comment:
Note that there is a `LinkWrapper` component in `app/routes/circulars.$circularId.($version)/Body.tsx` with which we replace all Markdown links. Please just set your new `isExternal` or `external` prop in that component, then you can undo this change:

```suggestion
        h('a', { target: '_blank' }, href),
```

Reply:
ah, I missed that, thanks!